sdk/communication/azure-communication-administration/tests/test_phone_number_administration_client.py

```diff
@@ -12,15 +12,23 @@
     NumberUpdateCapabilities,
     CreateSearchOptions
 )
+from azure.core.credentials import AccessToken
 from phone_number_helper import PhoneNumberUriReplacer
 from phone_number_testcase import PhoneNumberCommunicationTestCase
 from _shared.testcase import BodyReplacerProcessor
+from azure.identity import DefaultAzureCredential
+from azure.communication.administration._shared.utils import parse_connection_str
 
 SKIP_PHONE_NUMBER_TESTS = True
 PHONE_NUMBER_TEST_SKIP_REASON= "Phone Number Administration live tests infra not ready yet"
 
-class PhoneNumberAdministrationClientTest(PhoneNumberCommunicationTestCase):
+class FakeTokenCredential(object):
```

Comment:
I see this test and the async version of the test both use this class `FakeTokenCredential`,  can we move this class into common code